sdk/textanalytics/test-resources.json

```diff
@@ -0,0 +1,43 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "baseName": {
+            "type": "String"
+        },
+        "tenantId": {
+            "type": "String"
+        },
+        "testApplicationOid": {
+            "type": "String"
+        },
+        "endpointSuffix": {
+            "defaultValue": "cognitiveservices.azure.com",
+            "type": "String"
+        }
+    },
+    "variables": {
+        "location": "[resourceGroup().location]",
```

Comment:
I believe @mssfang mentioned that East US needs to be used, do you know what location this will be? I think it may be West US or West US2 by default, @hemanttanwar could you verify.

Reply:
We used to be told that West US/West US 2 is not functioning properly.  It is a service bug. But currently, the bug exists for other regions as well. So the region should not matter in this case.